include/aws/s3/private/s3_request.h

```diff
@@ -86,6 +87,9 @@ struct aws_s3_request {
 
     /* When true, this request is being tracked by the client for limiting the amount of in-flight-requests/stats. */
     uint32_t tracked_by_client : 1;
+
+    /* When true, even when the meta request is finished, this request will be sent. */
```

Comment:
`is finished` is a little confusing. Maybe `has finish result`?
Since when the abort is in process, we are not at the finished state, right?